### ISSUE TITLE
MUSIC

### ISSUE THREAD

**Issue by KENPOKASMMA1982** [TrieKai/KKBOX-CRM#1]
HELLO MATE!TELL ME PLEASE THIS PROJECT IS FOR SEARCHING MUSIC FROM KKBOX?

**Comment by TrieKai** [comment]
No, this is using the YouTube Data API to get data from the YouTube channel of KKKBOX.

**Comment by KENPOKASMMA1982** [comment]
woowwwwwww!!!mate could you please increase unlimited results for every song and album?It would be great my friend!

**Comment by TrieKai** [comment]
Sorry, each resource has a quota limit, you can check it [here](https://developers.google.com/youtube/v3/determine_quota_cost).

**Comment by KENPOKASMMA1982** [comment]
my friend do you think there is no any way to increase unlimited results?

**Comment by TrieKai** [comment]
I have no idea, do you have any solution?

**Comment by KENPOKASMMA1982** [comment]
really I dont know my friend!but this can help you??? but is for itunes!!!

I found a way to query more records. There is a parameter called offset, which is not mentioned in API documentation.

https://itunes.apple.com/search?term=a&offset=25&limit=25

https://stackoverflow.com/questions/44177089/itunes-search-api-page-number-for-the-query
